[ao] Added ModelReportVisualizer info to README for ModelReport

vspenubarthi opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #82796

Summary: This adds information on how the ModelReportVisualizer
integrates into the ModelReport API into the README file for the
ModelReport folder. It updates the high level usage flow, includes
information on the API, some of the important public methods and what
they do, as well as updates to the folder structure to include the new
`model_report_visualizer.py` file as well as updating the tests section
to highlight that there are high level tests for the
ModelReportVisualizer as well.

There really aren't any direct tests for this since it's just updates to
a README, but the tests for the ModelReportVisualizer are relavent and
were run to make sure table generation was still properly occuring.

Test Plan: python test/test_quantization.py TestFxModelReportVisualizer

Reviewers:

Subscribers:

Tasks:

Tags:

vspenubarthi committed:
[ao] Added ModelReportVisualizer info to README for ModelReport

Summary: This adds information on how the ModelReportVisualizer
integrates into the ModelReport API into the README file for the
ModelReport folder. It updates the high level usage flow, includes
information on the API, some of the important public methods and what
they do, as well as updates to the folder structure to include the new
`model_report_visualizer.py` file as well as updating the tests section
to highlight that there are high level tests for the
ModelReportVisualizer as well.

There really aren't any direct tests for this since it's just updates to
a README, but the tests for the ModelReportVisualizer are relavent and
were run to make sure table generation was still properly occuring.

Test Plan: python test/test_quantization.py TestFxModelReportVisualizer

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented on torch/ao/quantization/fx/_model_report/README.md:
````diff
@@ -21,25 +21,28 @@ This snippet should be ready to copy, paste, and use with the exception of a few
 ```python
 # prep model
-q_config_mapping = torch.ao.quantization.get_default_qconfig_mapping() # alternatively use your own qconfig mapping if you alredy have one
+q_config_mapping = torch.ao.quantization.get_default_qconfig_mapping()
````
nit: `q_config_mapping` -> `qconfig_mapping`

vspenubarthi committed:
Update on "[ao] Added ModelReportVisualizer info to README for ModelReport"


Summary: This adds information on how the ModelReportVisualizer
integrates into the ModelReport API into the README file for the
ModelReport folder. It updates the high level usage flow, includes
information on the API, some of the important public methods and what
they do, as well as updates to the folder structure to include the new
`model_report_visualizer.py` file as well as updating the tests section
to highlight that there are high level tests for the
ModelReportVisualizer as well.

There really aren't any direct tests for this since it's just updates to
a README, but the tests for the ModelReportVisualizer are relavent and
were run to make sure table generation was still properly occuring.

Test Plan: python test/test_quantization.py TestFxModelReportVisualizer

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

vspenubarthi commented:
@pytorchbot merge -g
